strawberry/experimental/pydantic/fields.py

```diff
@@ -79,3 +83,42 @@ def get_basic_type(type_) -> Type[Any]:
         return new_type_supertype(type_)
 
     return type_
+
+
+def replace_pydantic_types(type_: Any, is_input: bool):
+    if isinstance(type_, type):
+        if issubclass(type_, BaseModel):
```

Comment:
need to do `isinstance(type_, type)` like how we do it in get_basic_type otherwise newtypes cause an exception when you check `issubclass`. Because newtypes are a function <= python 3.10

Reply:
Can you make this an in-source comment?

Reply:
Can you make this an in-source comment?